.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "AlmaLinux_WWW_Container",
+  "build": {
+    "dockerfile": "Dockerfile",
+    "args": {
+      "VARIANT": "hugo",
+      "VERSION": "latest"
```

Comment:
as a note: I'm not sure the pinning you mentioned is actually working as intended. I recently (maybe as many as a few months ago) hit some snags due to outdated hugo installed on my laptop. I had to upgrade so that what I was doing would match the errors I was seeing as it deployed to dev.

Reply:
I think you're correct @bennyvasquez. The latest run I can see in GitHub actions uses the latest version 0.140.0

![image](https://github.com/user-attachments/assets/545cfc77-e775-4d64-a115-0923ed3ebb61)